yoda/pom.xml

```diff
@@ -0,0 +1,17 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>life.jukebox</groupId>
```

Comment:
Could you actually change groupId to life.jukebox.yoda

Reply:
This shouldn't break anything (other than the last arg to your entrypoint now needing to be "life.jukebox.yoda.App")